docs/localization-translation/README.md

```diff
@@ -0,0 +1,181 @@
+# Translating WooCommerce
+
+WooCommerce is already translated into several languages and is translation-ready right out of the box. All that’s needed is a translation file for your language.
+
+There are several methods to create a translation, most of which are outlined in the WordPress Codex. In most cases you can contribute to the project on [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/woocommerce/).
+
+To create custom translations:
+
+-   For a simple approach, choose the [Loco Translate](https://wordpress.org/plugins/loco-translate/) plugin.
+-   If you're more experienced, consider [Poedit ](https://poedit.net/).
+
+This document provides details on both methods.
+
+## Set up WordPress in your language
+
+To set your WordPress site's language:
+
+1. Go to `WP Admin » Settings » General` and adjust the `Site Language`.
+2. Go to `WP Admin » Dashboard » Updates` and click the `Update Translations` button.
+
+Once this has been done, the shop displays in your locale if the language file exists. Otherwise, you need to create the language files (process explained below).
+
+## Contributing your localization to core
+
+We encourage contributions to our translations. If you want to add translated strings or start a new translation, simply register at WordPress.org and submit your translations to [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/woocommerce/) for approval.
+
+## Translating WooCommerce into your language
+
+Both stable and development versions of WooCommerce are available for translation. When you install or update WooCommerce, WordPress will automatically fetch a 100% complete translation for your language. If such a translation isn't available, you can either download it manually or contribute to complete the translation, benefiting all users.
+
+If you’re new to translating, check out the [translators handbook](https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/) to get started.
+
+### Downloading translations from translate.wordpress.org manually
+
+1. Go to [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/woocommerce) and look for your language in the list.
+2. Click the title to be taken to the section for that language.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/2016-02-17-at-09.57.png)
+
+3. Click the heading under `Set/Sub Project` to view and download a Stable version.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/2016-02-17-at-09.59.png)
+
+4. Scroll to the bottom for export options. Export a `.mo` file for use on your site.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/2016-02-17-at-10.00.png)
+
+5. Rename this file to `woocommerce-YOURLANG.mo` (e.g., Great Britain English should be `en_GB`).
```

Comment:
Would be good to link to the acceptable list of language codes.

Reply:
I've added steps that explain how to locate the language code directly on the language pages, e.g. on https://translate.wordpress.org/locale/en-gb/default/wp-plugins/woocommerce/, and added the following screenshot:

<img width="371" alt="Screenshot 2023-10-17 at 09 44 53" src="https://github.com/woocommerce/woocommerce/assets/3323310/0cb21e53-07b7-4e7b-bd07-a770f2a11933">